deploy.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+set -u
+set -o pipefail
+
+readonly __SCRIPT_NAME__="${0##*/}"
+readonly __SEE_HELP_MESSAGE__="See '${__SCRIPT_NAME__} --help' for more information."
+
+function die {
+	local message="${1}"
+
+	printf "ERROR: %s\\n" "${message}" >&2
+
+	exit 1
+}
+
+function show_help {
+	cat << EOF
+${__SCRIPT_NAME__}"
+Deploy site to GitHub Pages branch
+Options:
+	-h, --help                    Show help text
+	-s, --source <branch>         Upstream branch (defaults to 'master')
+	-d, --destination <branch>    Branch to deploy to (defaults to 'gh-pages')
+	-r, --remote <remote>	        Remote to deploy to (defaults to 'origin')
+EOF
+}
+
+function is_working_tree_dirty {
+  ! git diff --quiet
+}
+
+# for gh-pages, use unpkg.com for dependencies
+function replace_onboarding_src {
+  local local_src="node_modules/@metamask/onboarding/dist/metamask-onboarding.bundle.js"
+  local web_src="unpkg.com/metamask-onboarding"
+  local target_file="index.html"
+
+  sed -i "" -e "s/${local_src}/${web_src}/g" "${target_file}"
+}
+
+function main {
+
+  local gh_remote='origin'
+  local source_branch='master'
+  local deploy_branch='gh-pages'
+
+	while :; do
```

Comment:
This file appears to have a mix of tabs and spaces 🤔

Reply:
Replaced with spaces